charts/posthog/templates/events-deployment.yaml

```diff
@@ -128,13 +128,24 @@ spec:
         readinessProbe:
           failureThreshold: {{ .Values.web.readinessProbe.failureThreshold }}
           httpGet:
-            path: /_health/
+            # For readiness, we don't want to bring down a pod if it can't query
+            # postgres. I this case we want to remain up, and still push to kafka
+            path: /_readyz/?exclude=postgres&exclude=postgres_migrations_uptodate
```

Comment:
@guidoiaquinti I thought `exclude` was an ok solution here, but I don't think it is now. Having the chart need to know the internals seems brittle.

I was thinking instead just have the chart know which service/role we are, i.e. pass in `role=events` then have the app decide what is important in this context.